data/campaigns/Under_the_Burning_Suns/units/undead/Spider_Lich.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=null
     cost=62
     usage=fighter
-    description= _ "No one is quite sure how spider liches are created, but they are a horrific sight to behold. Crawling around on huge skeletal legs, and wielding huge magical staves, these abominations are fearsome undead foes. "
+    description= _ "No one is quite sure how Spider Liches are created, but they are a horrific sight to behold. Crawling around on huge skeletal legs, and wielding huge magical staves, these abominations are fearsome undead foes. "
```

Comment:
Not related to this change, but could have trimmed the trailing space here.

Also, insects don't have internal skeletons, so how spider could become a skeleton is a mystery indeed!